extensions/douban/src/search-movies.tsx

```diff
@@ -111,6 +112,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${movie.title}(豆瓣)](${movie.url})`}
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.